oe_multilingual.install

```diff
@@ -16,56 +16,61 @@ use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationSelected;
+      ->save();
 
-  // Make sure that English language prefix is set to "en".
-  \Drupal::configFactory()
-    ->getEditable('language.negotiation')
-    ->set('url.prefixes.en', 'en')
-    ->save();
+    // Configure administrative language negotiation method.
+    \Drupal::configFactory()
+      ->getEditable('administration_language_negotiation.negotiation')
+      ->set('paths', [
+        '/admin',
+        '/admin/*',
+        '/node/add/*',
+        '/node/*/edit',
+        '/node/*/translations',
+      ])->save();
 
-  // Make sure that Portuguese language prefix is set to "pt".
-  \Drupal::configFactory()
-    ->getEditable('language.negotiation')
-    ->set('url.prefixes.pt-pt', 'pt')
-    ->save();
+    // Make sure that English language prefix is set to "en".
+    \Drupal::configFactory()
+      ->getEditable('language.negotiation')
+      ->set('url.prefixes.en', 'en')
+      ->save();
 
-  /** @var \Drupal\oe_multilingual\LanguageNegotiationSetterInterface $setter */
-  $setter = \Drupal::service('oe_multilingual.language_negotiation_setter');
+    // Make sure that Portuguese language prefix is set to "pt".
+    \Drupal::configFactory()
+      ->getEditable('language.negotiation')
+      ->set('url.prefixes.pt-pt', 'pt')
+      ->save();
 
-  // Set default language negotiation methods.
-  $setter->enableNegotiationMethods([
-    LanguageInterface::TYPE_INTERFACE,
-    LanguageInterface::TYPE_CONTENT,
-  ]);
+    /** @var \Drupal\oe_multilingual\LanguageNegotiationSetterInterface $setter */
+    $setter = \Drupal::service('oe_multilingual.language_negotiation_setter');
 
-  // For interface negotiation make sure administrative pages are in English.
-  $setter->setInterfaceSettings([
-    LanguageNegotiationAdministrationLanguage::METHOD_ID => -20,
-    LanguageNegotiationUrl::METHOD_ID => -19,
-    LanguageNegotiationSelected::METHOD_ID => 20,
-  ]);
+    // Set default language negotiation methods.
+    $setter->enableNegotiationMethods([
+      LanguageInterface::TYPE_INTERFACE,
+      LanguageInterface::TYPE_CONTENT,
+    ]);
 
-  // For content negotiation make sure that content respects URL language.
-  $setter->setContentSettings([
-    LanguageNegotiationUrl::METHOD_ID => -19,
-    LanguageNegotiationSelected::METHOD_ID => 20,
-  ]);
+    // For interface negotiation make sure administrative pages are in English.
+    $setter->setInterfaceSettings([
+      LanguageNegotiationAdministrationLanguage::METHOD_ID => -20,
+      LanguageNegotiationUrl::METHOD_ID => -19,
+      LanguageNegotiationSelected::METHOD_ID => 20,
+    ]);
+
+    // For content negotiation make sure that content respects URL language.
+    $setter->setContentSettings([
+      LanguageNegotiationUrl::METHOD_ID => -19,
+      LanguageNegotiationSelected::METHOD_ID => 20,
+    ]);
+
+    \Drupal::messenger()->addMessage('Please be aware that oe_multilingual makes changes to the following configurations: Locale, Administration language negotiation.');
```

Comment:
There is a change to "language.negotiation" settings, so module language is also affected. It should be listed.

Reply:
Will do, thanks!